services/movie_session.py

```diff
@@ -0,0 +1,44 @@
+from db.models import MovieSession
+
+
+def create_movie_session(
+        movie_show_time: int,
+        movie_id: id,
+        cinema_hall_id: int
+) -> None:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie_id=movie_id,
+        cinema_hall_id=cinema_hall_id
+    )
+
+
+def get_movies_sessions(session_date: None = None) -> None:

```

Comment:
The function `get_movies_sessions` should have a return type of `QuerySet` instead of `None`. Update the function signature to `def get_movies_sessions(session_date: Optional[date] = None) -> QuerySet:`.